java/test/server/commands/moves/OfferTradeCommandTest.java

```diff
@@ -23,8 +32,21 @@ public void tearDown() {
 
     }
 
+    /**
+     * checks to see if the command makes it to the facade and back
+     */
     @Test
     public void testExecute() {
-
+        List<ResourceType> send = new ArrayList<>();
+        List<ResourceType> receive = new ArrayList<>();
+        TradePackage package1 = new TradePackage(0,send);
+        TradePackage package2 = new TradePackage(1,receive);
+        Trade trade = new Trade(package1,package2);
+        OfferTradeDTO dto = new OfferTradeDTO(0,trade,1);
+        try {
+            CommandExecutionResult answer = Config.facade.offerTrade(1,dto);
+        } catch (OfferTradeException e) {
+            e.printStackTrace();
+        }
     }
```

Comment:
Can you add a few more test cases? This only tests the straight-forwarded expected norm case. Please throw in some `null`s, negative numbers, big numbers, etc. to thoroughly test the logic.